quarterly_reporting.py

```diff
@@ -251,19 +290,32 @@ def main():
         aggfunc="sum",
     )
 
+    # Gather spend plan data (originally in Google Drive)
     spend_plan = get_data(client, SPEND_PLAN)
+    # unit_code is used to join in the spend plan data, but after TPW merger we need to include department as well.
+    spend_plan["unit_code"] = spend_plan["department"].astype(str) + spend_plan["unit_code"]
+
     spend_plan["month_no"] = spend_plan.apply(encode_months, axis=1)
+    # Make the same month_col, so we can match it back to the expenses data
     spend_plan["month_col"] = (
         spend_plan["fiscal_year"].astype(str) + spend_plan["month_no"]
     )
 
+    # Merging the spend plan into the expenses for current quarter
     exp = to_output(exp, spend_plan)
+    # Creating column for selecting the quarter in Power BI
     exp["quarter_selection"] = f"{curr_quarter[:4]} Q{curr_quarter[-1:]}"
+
+    # Merging the spend plan into the expenses for previous quarter
     prev_exp = to_output(prev_exp, spend_plan)
+    # Creating column for selecting the quarter in Power BI
     prev_exp["quarter_selection"] = f"{prev_quarter[:4]} Q{prev_quarter[-1:]}"
 
+    # Append the previous expenses to the current expenses
     output = exp.append(prev_exp)
 
+    # Replace data in Socrata
+    output.drop(["department"], axis=1, inplace=True)
```

Comment:
department data ends up being encoded in the `unit_code` column. Like: 2507D001. 2507 is ATD department and D001 is unit bikeways. Power BI needs this column in order to join the metadata table to the expenses table.